README.md

```diff
@@ -1,6 +1,8 @@
 # Google Cloud Platform OpenJDK Docker Image
 
-This repository contains the source for the `gcr.io/google_appengine/openjdk` [docker](https://docker.com) image. This image can be used as the base image for running Java applications on [Google App Engine Flexible Environment](https://cloud.google.com/appengine/docs/flexible/java/) and [Google Container Engine](https://cloud.google.com/container-engine).
+This repository contains the source for the Google-maintained OpenJDK [docker](https://docker.com) image. This image can be used as the base image for running Java applications on [Google App Engine Flexible Environment](https://cloud.google.com/appengine/docs/flexible/java/) and [Google Container Engine](https://cloud.google.com/container-engine).
+
+This image can be found at `l.gcr.io/google/openjdk8`, or at `gcr.io/google-appengine/openjdk` if you're building an application to deploy on Google App Engine Flex.
```

Comment:
@gregw I think you're right about the wording, I went ahead and changed it to make it a bit more clear that the image is the same in both places. @meltsufin looks like we decided to go with the first one (I checked and `l.gcr.io/google/openjdk` doesn't exist).